CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Prowide ISO 20022 - CHANGELOG
 
+#### 9.3.6-SNAPSHOT
+  * (PW-631) Added a Charset parameter in the MxReadParams used by the MxParseUtils#createFilteredSAXSource to set the InputSource encoding
```

Comment:
Seria mas bien: 

Changed the XML marshaling/unmarshalling to honor the default system encoding, and added optional parameters to use a specific charset if needed

Reply:
Ok